packages/template-retail-react-app/package.json

```diff
@@ -40,10 +40,10 @@
     "@lhci/cli": "^0.11.0",
     "@loadable/component": "^5.15.3",
     "@peculiar/webcrypto": "^1.4.2",
-    "@salesforce/commerce-sdk-react": "3.1.0-dev",
-    "@salesforce/pwa-kit-dev": "4.0.0-dev",
-    "@salesforce/pwa-kit-react-sdk": "4.0.0-dev",
-    "@salesforce/pwa-kit-runtime": "4.0.0-dev",
+    "@salesforce/commerce-sdk-react": "3.1.1-extensibility-preview.0",
```

Comment:
Since we didn't change anything with this package does it even need a version bump? I guess it's simply to just bump all the independent packages.. but its a good question to ask and talk about.

Reply:
The change in `@salesforce/commerce-sdk-react` is bumping some dependencies to major versions. I think we are good bumping to `@salesforce/commerce-sdk-react` with a patch version to release the change.